resources/prepare-next-development-version.sh

```diff
@@ -21,7 +22,7 @@ git commit -am "release(v${RELEASE_VERSION})"
 git tag -fa "${RELEASE_VERSION}" -m "release(v${RELEASE_VERSION})"
 git push origin --tags -f
 
-mvn -B org.apache.maven.plugins:maven-release-plugin:update-versions -DgenerateBackupPoms=false
+mvn -B ${MAVEN_ADDITIONAL_OPTIONS} org.apache.maven.plugins:maven-release-plugin:update-versions -DgenerateBackupPoms=false
```

Comment:
please add this to silence the linter

```suggestion
# shellcheck disable=SC2086 # don't use quotes because we do want argument splitting
mvn -B ${MAVEN_ADDITIONAL_OPTIONS} org.apache.maven.plugins:maven-release-plugin:update-versions -DgenerateBackupPoms=false
```

Reply:
Added it, now the linter does not complain anymore.